lib/pf9-builder/testcase-03-ubuntu.csv

```diff
@@ -0,0 +1,10 @@
+## kubernetes
+# <hostname>|<image>:<username>|<flavor>|<network[,network,...]>|<key>|<security-group>|<role[,role]>[|<cluster-uuid>]
+k8s01|ubuntu-16.04-cloudimg-amd64-20180427:ubuntu|m1.medium|5ce89f0f-16aa-447a-a4db-d2910499d5af||cm-master|default|worker
+k8s02|ubuntu-16.04-cloudimg-amd64-20180427:ubuntu|m1.medium|5ce89f0f-16aa-447a-a4db-d2910499d5af||cm-master|default|worker
+k8s03|ubuntu-16.04-cloudimg-amd64-20180427:ubuntu|m1.medium|5ce89f0f-16aa-447a-a4db-d2910499d5af||cm-master|default|worker
+k8s04|ubuntu-16.04-cloudimg-amd64-20180427:ubuntu|m1.medium|5ce89f0f-16aa-447a-a4db-d2910499d5af||cm-master|default|worker
+k8s05|ubuntu-16.04-cloudimg-amd64-20180427:ubuntu|m1.medium|5ce89f0f-16aa-447a-a4db-d2910499d5af||cm-master|default|worker
+k8s06|ubuntu-16.04-cloudimg-amd64-20180427:ubuntu|m1.medium|5ce89f0f-16aa-447a-a4db-d2910499d5af||cm-master|default|worker
+k8s07|ubuntu-16.04-cloudimg-amd64-20180427:ubuntu|m1.medium|5ce89f0f-16aa-447a-a4db-d2910499d5af||cm-master|default|worker
+k8s08|ubuntu-16.04-cloudimg-amd64-20180427:ubuntu|m1.medium|5ce89f0f-16aa-447a-a4db-d2910499d5af||cm-master|default|worker
```

Comment:
this should be pulled out, extra stuff a customer cannot use but we can share amongst ourselves